COPYING

```diff
@@ -5,23 +5,41 @@ Source License, version 1.0, or at your option, any later version ("BOSL"). See
 the file ./LICENSE-BOSL for the terms of the Bootstrap Open Source Licence,
 version 1.0.
 
+In this document,
+
+- "Zcash" means the blockchain defined by the most recent agreement between
+  the Electric Coin Company and the Zcash Foundation according to section 6.2 (b)
+  of the Zcash Trademark Agreement.
+- A "Zcash Chain Fork" means a blockchain that descends from the Zcash
+  blockchain and that is or was forked within 3456 blocks of the current block
+  height of the Zcash blockchain at the time of the fork;
```

Comment:
I think that "at the time of the fork" could be further clarified to reference the divergence of the consensus rules; after all, if the consensus rules don't diverge, it's not actually a fork. This would eliminate any confusion that might be related to the time at which the *source code* was forked.

Reply:
Can you suggest a wording change? I have COVID and I'm flagging a bit (don't worry I'll be fine).